tests/installation/user_settings.pm

```diff
@@ -35,12 +35,19 @@ sub run {
     }
 
     $self->enter_userinfo();
-    assert_screen 'inst-userinfostyped';
-    if (match_has_tag 'boo#1122804') {
-        diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
-        $self->enter_userinfo(retry => 1);
-        assert_screen 'inst-userinfostyped';
+    assert_screen 'inst-userinfostyped-ignore-full-name';
```

Comment:
I consider this unnecessarily complicated and I fear some test maintainers would still not understand how to create a needle properly. It should be pretty clear from the old test code that the workaround code is only executed when the tag "boo#1122804" is found. The generic needle tag "inst-userinfostyped" is IMHO clear enough to understand.